src/cumulativeLine.ts

```diff
@@ -2,12 +2,13 @@ import * as d3 from "d3";
 import { Pareto } from "./pareto";
 import { moduleCategoryAxis, moduleIndices, modulePercentageAxis } from "./axis";
 import { resources } from "./resources";
+import { Settings } from "./settings";
 
 /**
  * Render the cumulative line using d3
  * @param pareto Pareto data structure
  */
-export function renderCumulativeLine(pareto: Pareto) {
+export function renderCumulativeLine(pareto: Pareto, settings: Settings) {

```

Comment:
Since we use a fixed color now you can remove the settings parameter and the import